### ISSUE TITLE
Can see tabs, can't see content

### ISSUE THREAD

**Issue by thely** [nickpeihl/leaflet-sidebar-v2#38]
Environment: I'm working in Node, using browserify to bundle all the JS files/libraries into one file, working in localhost, leaflet has been working fine thus far. All the CSS files are loading properly, and the JS is functioning, because I can see the `active` class get passed around when I click on tabs.

Issue: the actual panels of the sidebar are never visible. When a tab is selected, I can see that its content is present and active in the devtools inspector, but it looks like it's got opacity: 0 or it's stuck behind the map or something. I've tried this in both Firefox and Chrome. All the example projects work in my browser, though I haven't tried them through my localhost server. *Edit* to mention: I've tried both having the sidebar div inside the map div and having them separate. It doesn't make a difference.

Relevant code:

`index.html`
```html
<html>
	<head>
		<link rel="stylesheet" href="node_modules/leaflet/dist/leaflet.css">
		<link rel="stylesheet" href="node_modules/leaflet-sidebar-v2/css/leaflet-sidebar.min.css">
		<!--[if lte IE 8]>
			<link rel="stylesheet" href="node_modules/leaflet/dist/leaflet.ie.css" />
		<![endif]-->
		<link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.1.0/css/font-awesome.min.css" rel="stylesheet">
		<link rel="stylesheet" href="assets/css/style.css" />
	</head>
	<body>
		<div id="map">
			<div id="leaf-sidebar"></div>
		</div>
		<script src="./assets/js/bundle.js"></script>
	</body>
</html>
```
important parts of `bundle.js`:
```js
let sidebar = L.control.sidebar({
	position: 'left',
	container: 'leaf-sidebar'
}).addTo(map);

sidebar.addPanel({
	id: "home",
	tab: "<i class='fa fa-bars'></i>",
	title: "Home",
	pane: "This is a test home panel!",
}).open('home');

sidebar.addPanel({
	id: "test-box",
	tab: "T",
	title: "Test",
	pane: "<p>lorem ipsum etc</p>",
});
```

**Comment by thely** [comment]
Nevermind, figured it out - my sidebar had a different container ID, so the CSS wasn't working on it. Leaving this up in case someone else has the same issue!

**Comment by noerw** [comment]
Hi thely,
I can't reproduce your remark in your *edit* comment.
Using any value for the `container` option works for me. If it is a string, it's interpreted as DOM ID, and looked up (and when not found, a new container with that ID is created).

The issue seems be more of a missing `class="leaflet-sidebar"` attribute on your sidebar container.

Note that specifying a container in HTML in the first place is optional, and only useful if you have some static content to declare.

**Comment by thely** [comment]
I did finally get `container: 'name'` names to work with already-present `<div id='name'> after reading your comment, but only if I wrote the option as `container: '#name'`, which isn't how any of the documentation presents it. I'm not sure why that is, because all the rest of your examples don't include a hash.

**Comment by noerw** [comment]
Indeed, prepending a hash shouldn't be necessary.
Which leaflet version are you using? Could you provide a reproducible example?

**Comment by thely** [comment]
I'm using Leaflet v1.5.1 - worth noting that `L.map` in the most recent version doesn't require a hash, either. I tried putting together both a [jsfiddle](https://jsfiddle.net/mdgbf0kL/1/) and a [pen](https://codepen.io/anon/pen/EqmVmV) to see if I could replicate it, but I keep getting `L.control.sidebar() is not a function` in the console on both. (I had to add a `var L = require("leaflet");` to the top of the leaflet-sidebar source to get it to bundle with browserify, but that shouldn't matter here?)

I don't have an online/running version of my project yet, but [its repo is here](https://github.com/thely/shorebird-leaflet). I think the only thing that wouldn't work for you is probably the audio.
